Stratis.Bitcoin.Features.Miner/Controllers/MinerController.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Linq;
+using System.Net;
+using System.Security;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Features.Miner.Models;
+using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.Utilities.JsonErrors;
+
+namespace Stratis.Bitcoin.Features.Miner.Controllers
+{
+    /// <summary>
+    /// Controller providing operations on a wallet.
```

Comment:
this looks like a copy and paste comment

Reply:
Fixed